Mathlib/LinearAlgebra/QuadraticForm/Basic.lean

```diff
@@ -6,46 +6,47 @@ Authors: Anne Baanen, Kexing Ying, Eric Wieser
 import Mathlib.LinearAlgebra.Matrix.Determinant.Basic
 import Mathlib.LinearAlgebra.Matrix.SesquilinearForm
 import Mathlib.LinearAlgebra.Matrix.Symmetric
+import Mathlib.LinearAlgebra.BilinearMap
 
 #align_import linear_algebra.quadratic_form.basic from "leanprover-community/mathlib"@"d11f435d4e34a6cea0a1797d6b625b0c170be845"
 
 /-!
-# Quadratic forms
+# Quadratic maps
 
-This file defines quadratic forms over a `R`-module `M`.
-A quadratic form on a commutative ring `R` is a map `Q : M → R` such that:
+This file defines quadratic maps over a `R`-module `M`.
+A quadratic map on a commutative ring `R` is a map `Q : M → N` such that:
 
-* `QuadraticForm.map_smul`: `Q (a • x) = a * a * Q x`
-* `QuadraticForm.polar_add_left`, `QuadraticForm.polar_add_right`,
-  `QuadraticForm.polar_smul_left`, `QuadraticForm.polar_smul_right`:
-  the map `QuadraticForm.polar Q := fun x y ↦ Q (x + y) - Q x - Q y` is bilinear.
+* `QuadraticMap.map_smul`: `Q (a • x) = a * a * Q x`
```

Comment:
Should this be `a • a • Q x`  (or `(a * a) • Q x`) on the right?

Reply:
```suggestion
* `QuadraticMap.map_smul`: `Q (a • x) = (a * a) • Q x`
```